pkcs11/_pkcs11.pyx

```diff
@@ -1550,6 +1550,19 @@ cdef class lib:
         except StopIteration:
             return token
 
+    def wait_for_slot_event(self, blocking=False):
```

Comment:
I think the default is probably blocking?

Reply:
I'm not sure, but I'm not picky, I'll make the change

Reply:
It's the default in PKCS#11 itself (flags = 0).

Reply:
I see what you mean, the example is showing flag=0. I'll make that change.